lib/checks/lists/listitem.js

```diff
@@ -1,18 +1,18 @@
 const parent = axe.commons.dom.getComposedParent(node);
 if (!parent) {
-	return false;
+	// Can only happen with detached DOM nodes and roots:
+	return undefined;
 }
 
-const ALLOWED_TAGS = ['UL', 'OL'];
 const parentTagName = parent.nodeName.toUpperCase();
-
 const parentRole = (parent.getAttribute('role') || '').toLowerCase();
-if (parentRole && !axe.commons.aria.isValidRole(parentRole)) {
+
+if (parentRole === 'list') {
+	return true;
+}
+
+if (parentRole && axe.commons.aria.isValidRole(parentRole)) {
```

Comment:
Shouldn't this check for ARIA roles that aren't `list` to return false, like the code did before? 
```
if (parentRole && !axe.commons.aria.isValidRole(parentRole)) {
```